src/Neo.Cryptography.BLS12_381/Neo.Cryptography.BLS12_381.csproj

```diff
@@ -5,6 +5,8 @@
     <TargetFrameworks>netstandard2.1;net8.0</TargetFrameworks>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PackageId>Neo.Cryptography.BLS12_381</PackageId>
```

Comment:
Is this needed, what is this packageID need?

Reply:
is this package created here or elsewhere?

Reply:
I searched quickly here and found that packageId was more related to dependency.

Reply:
Its for `nuget`